spec/ruby/core/float/positive_spec.rb

```diff
@@ -0,0 +1,27 @@
+require_relative '../../spec_helper'
+
+describe "Float#positive?" do
+  describe "on positive numbers" do
+    it "returns true" do
+      0.1.positive?.should be_true
+    end
+  end
+
+  describe "on zero" do
+    it "returns false" do
+      0.0.positive?.should be_false
+    end
+  end
```

Comment:
Similarly, it would be interesting to test on `-0.0` too

Reply:
Done.